scripts/install_from_src_ubuntu

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euo pipefail
+
+clean_slate()
+{
+	echo "                      =============================================="
+	echo "                      If you are within a conda env please deactivate it and come back"
+	echo "                      If you don't want to proceed kill this script in the next 10 seconds!"
+	echo "                      =============================================="
+
+	sleep 10
+
+	echo "Cleaning up state"
+	# uninstall any global installations
+	pip uninstall --yes torchserve || true
+	pip uninstall --yes torch-model-archiver || true
+	sleep 5
+}
+
+install_java_deps()
+{
+	sudo apt-get install -y openjdk-11-jdk
+}
+
+install_torch_deps()
+{
+	# replace this with requirements.txt when ready
+	pip install torch torchtext torchvision sentencepiece psutil future
+}
+
+build_install_server()
+{
+	python setup.py build
+	pip install .
```

Comment:
This will run the build process twice.

Reply:
`pip install .` already dumps complete logs on console if the build fails.

Reply:
This was proposed by Vamshi from his prior background. Can you please check with him? Until then I will err on the side of caution (we can always remove if we're convinced that it's redundant)